### PR TITLE
OKTA-571051 Add new SAML customer config doc examples

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add new SAML customer config doc examples with metadata from the UI. Acrolinx fixes
- **Is this PR related to a Monolith release?** No

### Preview:

https://6424866c25f06411430f7d00--reverent-murdock-829d24.netlify.app/docs/guides/submit-app-prereq/main/#customer-configuration-document-guidelines

### Resolves:

* [OKTA-571051](https://oktainc.atlassian.net/browse/OKTA-571051)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/819a66f8c6ec0f37791c93c5e5374883d4a89ce3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/037c30df-aa99-420e-b391-6ff34ff86276) | 65  | 82  | 54  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.6fe7b9d1-9c1e-45a7-9fb8-92523b839ef2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/aa06d3a490601682401868fe55eb55d13eb1f497/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/66c71cb0-0042-4a19-9879-65f36175b8bf) | 65  | 82  | 54  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e96a1ec7-e4b4-4aab-aecc-5c4a54d893a3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/a7a6569999d4d9adfdcf24c2c34937625cddc1e6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ac564a2e-1747-4653-bf82-751670685ac6) | 65  | 82  | 54  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.14bd7e41-51a7-4e02-9112-c3830d3b4002?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/070e7adb3435fe4b8e34103d5c19bd2c24d730cc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/97143cb0-2602-4098-ac47-6bc582d026d4) | 65  | 82  | 63  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.83d19f97-69dd-4cde-9ad6-8e678534c684?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/f4b03d45a4df19b9868a57c9afecd0aea12260aa/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c6e4891a-8a54-44fd-b0df-ba6e1fc2a1c5) | 65  | 82  | 63  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md](https://github.com/okta/okta-developer-docs/blob/f4b03d45a4df19b9868a57c9afecd0aea12260aa/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app%2Fmain%2Fsaml2%2Fsubmit.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ff9706c4-2e7e-4969-8ee6-e1f402085988) | 75  | 46  | 54  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.fb625b36-4efb-4821-86cd-9e65a1b2be0e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/54f6422e47844e2a92e312a61b3966132d0f7cef/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6c807643-8ece-48b6-8b09-ff6a8a32c04d) | 65  | 82  | 63  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md](https://github.com/okta/okta-developer-docs/blob/54f6422e47844e2a92e312a61b3966132d0f7cef/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app%2Fmain%2Fsaml2%2Fsubmit.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/00f69a6e-db1f-4b45-86b2-802fac106892) | 75  | 46  | 54  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.5ea1ab86-5687-4e10-b8ff-085606737af8?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/f9f467b8b0e9414e7686bb7dbd1caf32b155cc3a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bfded3a8-abbc-4a1f-ad37-3174e45403c1) | 65  | 82  | 63  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md](https://github.com/okta/okta-developer-docs/blob/f9f467b8b0e9414e7686bb7dbd1caf32b155cc3a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app%2Fmain%2Fsaml2%2Fsubmit.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6228491a-075d-464d-af2e-760a40723396) | 75  | 46  | 54  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.20abf6c0-238f-465c-995d-98ab32fb8eb2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/b3cebf5f1411a849842ae4aa62e22ae7441a548f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f415b5fd-0498-4e84-915b-62007fca3947) | 65  | 83  | 62  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md](https://github.com/okta/okta-developer-docs/blob/b3cebf5f1411a849842ae4aa62e22ae7441a548f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app%2Fmain%2Fsaml2%2Fsubmit.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/60477d7c-21e8-4fc9-9b40-7e274ce1a35d) | 75  | 46  | 54  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.ff3f2bde-5af2-436e-85d9-3b692c57fd83?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/41feaa1f36b2d5a6b14b246e9e0ec574db9f4abb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4dda7a0f-0758-4f86-9c12-97052d2e51f2) | 66  | 80  | 62  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md](https://github.com/okta/okta-developer-docs/blob/41feaa1f36b2d5a6b14b246e9e0ec574db9f4abb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app%2Fmain%2Fsaml2%2Fsubmit.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/74299cec-a475-41b9-bc32-4f080a1c6a45) | 75  | 46  | 54  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.1d295924-887a-4ce4-a3a4-cd0ad3a24229?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:60]: Do we really need to offer the oin@ handle here? "If you have more logo questions, contact the [OIN team](mailto:oin@okta.com)."

Recommend either putting more detail around why ISVs would email further beyond this logo instruction, or just remove that sentence.

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:80]: I see Ops team very commonly sending ISVs examples of descriptions, can we add these examples? 

Here are some examples:
Infinipoint enables enterprises of all sizes to automate cyber hygiene across their entire IT estate with unprecedented speed and scale. For the benefit of single-sign-on, Okta's Infinipoint integration gives your team the ability to securely manage and secure the organisation's IT assets.
Five to Nine is a company event discovery, management and evaluation software. From single sign-on to user provisioning, Okta’s Five to Nine integration handles user access throughout the user’s lifecycle from onboarding to offboarding. The integration also allows you to import custom user attributes from Okta into Five to Nine.
Enables Edify customers to log into the Edify platform using Okta as a single sign-on provider`

These are what the Ops team email to ISVs as example  good descriptions.

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:131]: should this be `is accessible to the general public` ?

After Ops checks the config guide is good, I don't think they are going back after publishing ensure its public to anyone on the internet.

**Review by albertchen-okta — APPROVED** [review]
a few comments but approved to not hold you back

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:131]: Let me verify with Mohammad Abdullah about this point. This sentence was left in because some ISV might not want their doc public until it's been reviewed and approved.

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:80]: I talked with Mohammad Abdullah: he gave me a generic description to use. We don't have approval to use Infinipoint or Five to Nine descriptions.

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:131]: I talked with Mohammad Abdullah: leaving the blurb as-is. some ISVs publish their docs after their app has been published.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md](https://github.com/okta/okta-developer-docs/blob/6d764dc8822f65ebd68b25ea8f0e2ace84e72963/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app-prereq%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c73df4da-6351-48f3-afd6-f698f9584065) | 69  | 81  | 60  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md](https://github.com/okta/okta-developer-docs/blob/6d764dc8822f65ebd68b25ea8f0e2ace84e72963/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsubmit-app%2Fmain%2Fsaml2%2Fsubmit.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e5aecd0e-e984-4f54-a13c-2604807b8c54) | 91  | 46  | 53  | 84  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4e9b33d5-b86a-4a76-99be-7e8e171b6e46?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by vanngo-okta** [comment]
Mohammad Abdullah approved via Slack

**Comment by pvershinin-okta** [comment]
LGTM!

File: packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md:26]:
```suggestion
* **To configure SAML, can your customers do it by themselves from your app's UI, or do they need to contact your support team?**: If a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. See [SAML configuration steps example](/docs/guides/submit-app-prereq/main/#saml-configuration-steps-example) for configuration steps when the customer requires the support team to configure SAML. See [SAML admin configuration steps example](/docs/guides/submit-app-prereq/main/#saml-admin-configuration-steps-example) for configuration steps when the customer configures SAML for themselves. Okta recommends that you build a UI that enables self-service configuration to reduce the setup time for your customers.
```

**Review by susanharper-okta — APPROVED** [review]
couple nits - approving to not hold you up

**Pull request merged**
